cmd/brew-upgrade-mysql

```diff
@@ -22,6 +22,9 @@ install_mysql() {
     brew install mysql@${mysql_version}
   fi
 
+  # In case jq is not installed
+  brew install jq &> /dev/null
```

Comment:
`brew list jq &>/dev/null || brew install jq` will display output only when it's installed (so people know what's going on and any errors are shown) and avoid always failing when it's already installed.